src/www/ui/api/Controllers/UploadController.php

```diff
@@ -391,7 +391,7 @@ public function postUpload($request, $response, $args)
         $info = new Info(500, $message . "\n" . $statusDescription,
           InfoType::ERROR);
       } else {
-        $uploadId = $uploadResponse[3][0];
+        $uploadId = $uploadResponse[3];
```

Comment:
This change will cause issues for file uploads as the third argument returned is an array and clients expect integer
https://github.com/fossology/fossology/blob/1f490d46d79ae9ea85ff0de279083b271147bbc5/src/www/ui/page/UploadFilePage.php#L201-L202

To fix that, following location must be updated
https://github.com/fossology/fossology/blob/1f490d46d79ae9ea85ff0de279083b271147bbc5/src/www/ui/api/Helper/UploadHelper/HelperToUploadFilePage.php#L47

to something like this
```php
$response = $this->handleUpload($request);
$response[3] = $response[3][0];
return $response;
```